### DISCUSSION TITLE
PDF 출력 기능도 만들면 어떨까요?

### DISCUSSION THREAD

**Discussion by designe** [euphony-io/check-in-listener#24] [Ideas]
![image](https://user-images.githubusercontent.com/1748714/134792420-8152557d-84bc-49de-94b3-821ac386124e.png)
이런 느낌으로 pdf 출력 기능이 있다면 사업자 입장에서 편리하게 쓸 수 있겠다는 생각이 듭니다 ㅋㅋ

**Comment by Jsueeee** [comment]
pdf 가 사업자 입장에선 더 한 눈에 보일 것 같긴 합니다! 좋은 아이디어 감사합니다 😄
그럼 현재 진행하고 있는 csv 파일은 서버 전송용이고, pdf 파일 생성은 사업자 확인용?으로 나누어야할까요?

**Reply by designe** [reply]
그렇게 나눌수 있을 것 같네요!
한번 검토해보고 부담이 되면 안넣어도 상관없을거 같아요ㅋㅋㅋ

**Reply by Jsueeee** [reply]
아마도 기간을 고려해야할 것 같아요 ㅎㅎㅎ
멘토님이 말씀하셨던 컨택 전에 맞춰서 완성을 해야할 것 같은데,
10월 초안에 스피커랑 리스너 앱 모두 완전히 마무리를 해야겠죠...??

**Reply by yeonns** [reply]
@Jsueeee 빡세지만 이번주안에 마무리하는걸 목표로 도전해봤으면 좋겠어요! 만약 시간이 더 필요하다면 한주더...!? 😭

**Reply by Jsueeee** [reply]
이번 주 안에 마무리하도록 해보겠습니다!!🤣🤣
